docs/content/preview/migrate/performance.md

```diff
@@ -24,39 +24,41 @@ There are several factors that slow down data-ingestion performance in any datab
-\
-  Use the [-–parallel-jobs](../yb-voyager-cli/#parallel-jobs) argument with the import data command to override the default setting. Set --parallel-jobs to the number of available cores in the entire cluster.\
-\
-  If CPU use is greater than 80%, you should lower the number of jobs. Similarly, if CPU use is low, you can increase the number of jobs.
+- **Load data in parallel**. yb-voyager executes N parallel batch ingestion jobs at any given time, where N is equal to half of the total number of cores in the YugabyteDB cluster. Normally this is considered good default practice. However, if the target cluster runs on high resource machines with a large number of CPU cores, the default may result in underusing CPU resources.
 
-- **Increase batch size**. If the [--batch-size](../yb-voyager-cli/#batch-size) (default is 100000) is too small, the import will run slower because the time spent importing data may be comparable or less than the time spent on other tasks, such as bookkeeping, setting up the client connection, and so on.
+  Use the [-–parallel-jobs](../yb-voyager-cli/#parallel-jobs) argument with the import data command to override the default setting. Set `--parallel-jobs` to a number higher than the default based on your cluster configuration.
 
-- **Add disks** to reduce disk write contention. YugabyteDB servers can be configured with one or multiple disk volumes to store tablet data. If all tablets are writing to a single disk, write contention can slow down the ingestion speed. Configuring the [YB-TServers](../../reference/configuration/yb-tserver/) with multiple disks can reduce disk write contention, thereby increasing throughput. Disks with higher IOPS and better throughput also improve write performance.
+  If CPU use is greater than 50-60%, you should lower the number of jobs. Similarly, if CPU use is low, you can increase the number of jobs.
+
+   {{< note title="Note" >}}
+
+   If there are timeouts during import, restart the import with fewer parallel jobs.
 
-- **Specify the number of table splits**:
+   {{< /note >}}
 
-  - For larger tables and indexes that are [hash](../../architecture/docdb-sharding/sharding/#hash-sharding) sharded, specify the number of initial tablet splits as a part of the table DDL statement. This can help distribute the table data across multiple nodes right from the get go. Refer to [hash-sharded tables](../../architecture/docdb-sharding/tablet-splitting/#hash-sharded-tables) for an example of how to specify the number of tablets at table creation time.
+- **Increase batch size**. If the [--batch-size](../yb-voyager-cli/#batch-size) (default is 20000) is too small, the import will run slower because the time spent importing data may be comparable or less than the time spent on other tasks, such as bookkeeping, setting up the client connection, and so on. Increasing the batch-size to a very high value is not recommended as the whole batch is executed in one transaction.
```

Comment:
**Existing**

> Increase batch size. If the [--batch-size](https://deploy-preview-15688--infallible-bardeen-164bc9.netlify.app/preview/migrate/yb-voyager-cli/#batch-size) (default is 20000) is too small, the import will run slower because the time spent importing data may be comparable or less than the time spent on other tasks, such as bookkeeping, setting up the client connection, and so on. Increasing the batch-size to a very high value is not recommended as the whole batch is executed in one transaction.

**Proposed**

> Increase batch size. The default [--batch-size](https://deploy-preview-15688--infallible-bardeen-164bc9.netlify.app/preview/migrate/yb-voyager-cli/#batch-size) is 20000 or approximately 200 MB of data depending on whichever is reached first while preparing the batch. Normally this is a good default value. However if the rows are too small then one can think of increasing the batch size for greater throughput. Increasing the batch-size to a very high value is not recommended as the whole batch is executed in one transaction.